lib/libuutil/uu_avl.c

```diff
@@ -28,7 +28,11 @@
 #include "libuutil_common.h"
 
 #include <stdlib.h>
+#ifdef HAVE_LIBBSD
+#include <bsd/string.h>
+#else
 #include <string.h>
+#endif
```

Comment:
why do we need bsd/string.h in files that are otherwise unmodified?

Reply:
Good question. zfs.8 didn't need a date bump either. I'll clean up the patch a bit more and see if it still works.

Reply:
It wasn't immediately obvious, but the reason was that when the check for setproctitle finds it in libbsd, configure also finds strlcat and strlcpy there, so HAVE_STRLCAT and HAVE_STRLCPY get defined and the versions of those functions living in libspl do not get defined. This is then a problem unless bsd/string.h is included.

I have found a less invasive way to handle this.